LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
```

Comment:
Should this be a `.md`?

Reply:
According to GitHub, yes it should be! Renamed :rocket: